scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -1495,6 +1501,7 @@ Function ValidateTestParameters
         If ([string]::IsNullOrEmpty($SnitchAlertUrl)) {Throw "Required snith alert URL."}
         If ([string]::IsNullOrEmpty($SnitchStorageAccount)) {Throw "Required snitch storage account."}
         If ([string]::IsNullOrEmpty($SnitchStorageMasterKey)) {Throw "Required snitch storage master key."}
+        If ([string]::IsNullOrEmpty($HostPlatform)) {Throw "Required host platform."}
```

Comment:
Are you going to remove this line here and check only if test is stress or long haul, right?

Reply:
Wait, isn't that what it is currently doing?

Reply:
Sorry; the code is collapsed and confused me.